### PR TITLE
Predicted label needs to be of Key Type.

### LINKED ISSUE #3465: "Predicted label" type needs to be KeyType and not UInt32 in IO-Columns-Multiclass MD file
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.lightgbm.lightgbmmulticlasstrainer?view=ml-dotnet&branch=smoke-test-preview

### PR DESCRIPTION
**Pull request by codemzs**
fixes #3465

### CODE REVIEW

File: docs/api-reference/io-columns-multiclass-classification.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ This trainer outputs the following columns:
 | Output Column Name | Column Type | Description|
 | -- | -- | -- |
 | `Score` | vector of <xref:System.Single> | The scores of all classes. Higher value means higher probability to fall into the associated class. If the i-th element has the largest value, the predicted label index would be i. Note that i is zero-based index. |
-| `PredictedLabel` | <xref:System.UInt32> | The predicted label's index. If it's value is i, the actual label would be the i-th category in the key-valued input label type. Note that i is zero-based index. |
+| `PredictedLabel` | [key](xref:Microsoft.ML.Data.KeyDataViewType) type | The predicted label's index. If it's value is i, the actual label would be the i-th category in the key-valued input label type. Note that i is zero-based index. |
```
Comment (wschin) [inline docs/api-reference/io-columns-multiclass-classification.md]:
```suggestion
| `PredictedLabel` | [key](xref:Microsoft.ML.Data.KeyDataViewType) type | The predicted label's index. If it's value is i, the actual label would be the i-th category in the key-valued input label type. |
```

Comment (Ivanidzo4ka) [inline docs/api-reference/io-columns-multiclass-classification.md]: >key [](start = 22, length = 3)

Can we capitalize `[Key]` ?

Comment (codemzs) [reply docs/api-reference/io-columns-multiclass-classification.md]: we can but then we need to make this change everywhere. Right now everywhere it is "key" and it is consistent. What do you suggest?

File: docs/api-reference/io-columns-multiclass-classification.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ This trainer outputs the following columns:
 | Output Column Name | Column Type | Description|
 | -- | -- | -- |
 | `Score` | vector of <xref:System.Single> | The scores of all classes. Higher value means higher probability to fall into the associated class. If the i-th element has the largest value, the predicted label index would be i. Note that i is zero-based index. |
```
Comment (Ivanidzo4ka) [inline docs/api-reference/io-columns-multiclass-classification.md]: >vector [](start = 12, length = 6)

Shall we start sentences with capital letter?

File: docs/api-reference/io-columns-multiclass-classification.md (hunk 1 above)
Comment (natke) [reply docs/api-reference/io-columns-multiclass-classification.md]: it's --> its

Comment (natke) [reply docs/api-reference/io-columns-multiclass-classification.md]: it's --> its

File: docs/api-reference/io-columns-multiclass-classification.md (hunk 2 above)
Comment (codemzs) [reply docs/api-reference/io-columns-multiclass-classification.md]: Spoke offline with @Ivanidzo4ka  @shmoradims and got feedback from @natke that either "key" or "Key" is fine.

**Pull request merged**